### DISCUSSION TITLE
Initialise display options or trigger dropdown ?

### DISCUSSION THREAD

**Discussion by gg4u** [sagalbot/vue-select#1277] [Q&A] [answered]
I using the component in a modal, and when the modal opens, I'd like to prompt the user with a list of options already visible, without the user needing to focus on the bar.

I looked at the guide about events, but events are emitted, instead I would like the dropdown to listen to an event - or just make it visibile.

It is different behaviour than  :closeOnSelect="false" , because user is still to select an item.

Any possibility / workaround to achieve this behaviour already?

**Comment by djokicpn** [comment, marked as answer]
Just add ref to your component
`<v-select ref="dropdown"> </v-select>`

After that just use $refs: 
```
openModalFunction() {
    this.$nextTick(function () {
        this.$refs.dropdown.$refs.search.focus()
    });
}
```

This should help
